web documents/metsoverview_de.md

```diff
@@ -2,50 +2,16 @@
-aufbauend auf der Arbeit von MOA2 ein XML-Dokumentformat zu entwickeln,
-das kodierte Metadaten sowohl für die Verwaltung von Objekten einer
-digitalen Bibliothek in einer Sammlung wie auch für den Austausch
-solcher Objekte mit anderen Sammlungen (oder die Übermittlung an
-Benutzer) aufnehmen kann. Ein METS-Dokument kann zudem im Kontext des
-[Open Archival Information System
-(OAIS)-Referenzmodells.](http://ssdoo.gsfc.nasa.gov/nost/isoas/ref_model.html)
-\[für die langfristige Erhaltung digitaler Aufzeichnungen\] entweder als
-Liefereinheit (Submission Information Package - SIP),
-Archivierungseinheit (Archival Information Package - AIP), oder
-Bereitstellungseinheit (Dissemination Information Package - DIP)
-verwendet werden.
-
-Ein METS-Dokument besteht aus 7 Hauptabschnitten:
+METS ist ein Kodierungsstandard für beschreibende, administrative/technische und strukturelle Metadaten von komplexen digitalen Objekten unterschiedlichster Art, bspw. Text-, Bild- oder Multimedia-Objekte. Ursprünglich eine Initiative der [Digital Library Federation](http://www.diglib.org/) ist METS ein XML-Dokumentformat für die Kodierung von Metadaten sowohl für die Verwaltung von digtialen Objekten innerhalb eines digitalen Magazins als auch für den Austausch solcher Objekte zwischen den digitalen Magazinen und den Benutzern/Benutzerinnen. Abhängig vom jeweiligen Anwendungsfall können METS-Dokumente im Kontext des [Open Archival Information System (OAIS)-Referenzmodells.](http://ssdoo.gsfc.nasa.gov/nost/isoas/ref_model.html) \[für die langfristige Erhaltung digitaler Aufzeichnungen\] entweder als Liefereinheit (Submission Information Package - SIP), Archivierungseinheit (Archival Information Package - AIP), oder Bereitstellungseinheit (Dissemination Information Package - DIP) verwendet werden.
+
+METS bietet ein kohärentes, integriertes Modell für die komplexen Metadaten, die für die Pflege von einer Sammlung digitaler Objekte benötigt werden. Diese Metadaten sind sowohl ausführlicher als auch unterschiedlich im Vergleich zu den Informationen über Sammlungen von gedruckten Werken oder anderen physischen Objekten. Ein gedrucktes Buch bspw. wird weder in einzelne Blätter zerfallen wenn keine Strukturangaben über die innere Ordnung des Buches erfasst werden, noch werden Forscher das Buch schlechter nutzen können wenn Angaben über die Herstellungsart fehlen. Gleiches gilt jedoch nicht für eine digitale Version desselben Buches. Ohne Metadaten zur Struktur sind die Seitenabbildungen oder die Textdateien, aus denen es besteht, so gut wie wertlos. Und ohne technische Metadaten über den Digitalisierungsprozess können Leser nicht sicher sein, wie genau die digitale Version die ursprüngliche Vorlage wiedergibt. Eine Organisation muss auch notwendige technische Metadaten für interne Verwaltungszwecke verfügbar haben, um bspw. die Daten regelmässig überprüfen und migrieren zu können und damit langfristige Zugänglichkeit der wertvollen Quellen zu garantieren.
```

Comment:
METS bietet ein kohärentes, integriertes Framework für die komplexen Metadaten, die für die Pflege von einer Sammlung digitaler Objekte benötigt werden. Diese Metadaten sind sowohl ausführlicher als auch unterschiedlich im Vergleich zu denen über Sammlungen von gedruckten Werken oder anderen physischen Objekten. Ein Buch bspw. wird weder in einzelne Blätter zerfallen, wenn keine Strukturmetadaten über die innere Ordnung des Buches vom Besitzer erfasst werden, noch wird es Wissenschaftlern unmöglich sein, den Nutzungswert beurteilen zu können, wenn der Hersteller keine Angaben zum Entstehungsprozess gemacht hat. Gleiches gilt jedoch nicht für eine digitale Version desselben Buches. Ohne Strukturmetadaten sind die Seitenabbildungen oder die Textdateien, aus denen es besteht, so gut wie wertlos. Und ohne technische Metadaten über den Digitalisierungsprozess können Wissenschaftler nicht sicher sein, wie genau die digitale Version die ursprüngliche Vorlage wiedergibt. Eine Organisation muss auch über notwendige technische Metadaten für interne Verwaltungszwecke verfügen, um bspw. die Daten regelmäßig überprüfen und migrieren zu können und damit die dauerhafte Nutzbarkeit der wertvollen Ressourcen sicher zu stellen.

Reply:
Hallo @gewappnet 

Danke, hab's fast 1:1 so übernommen.